sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2362,6 +2362,10 @@ private AnalyzeBatchActionsOperation StartAnalyzeBatchActions(MultiLanguageBatch
             {
                 tasks.KeyPhraseExtractionTasks = Transforms.ConvertFromKeyPhrasesOptionsToTasks(actions.ExtractKeyPhrasesOptions);
             }
+            if (actions.RecognizeLinkedEntitiesOptions != null)
```

Comment:
the docstrings of the method (same with sync) need to be updated too to include entity linking. something like `multiple actions including entity recognition, PII entity recognition, linked entity recognition and key phrases extraction`.